src/blockchain/shelley/block.js

```diff
@@ -112,11 +120,16 @@ export default class ShelleyBlock implements Block {
       if (fragment.is_transaction()) console.log(`#${index} = TRANSACTION`)
       if (fragment.is_owner_stake_delegation()) console.log(`#${index} = OWNER STAKE DELEG`)
       if (fragment.is_stake_delegation()) console.log(`#${index} = STAKE DELEG`)
-      // if (fragment.is_pool_registration()) console.log('#' + index + ' = POOL REG')
       if (fragment.is_pool_retirement()) console.log(`#${index} = POOL MANAGE`)
       if (fragment.is_transaction() || fragment.is_owner_stake_delegation()
         || fragment.is_pool_registration() || fragment.is_stake_delegation()) {
-        txs.push(shelleyUtils.fragmentToObj(fragment, txCommon))
+        txs.push(shelleyUtils.fragmentToObj(fragment,
```

Comment:
Do we want this for `OldUtxoDeclaration` or do we not care since we always know it's in the genesis block?